README.md

```diff
@@ -107,7 +113,7 @@ Open mobile.html using live server extension.
 - GitHub: [@JuanDBta](https://github.com/JuanDBta)
 - Twitter: [@diazgjuan](https://twitter.com/diazgjuan)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/juan-david-diaz-)
-- Loom: [Loom](https://www.loom.com/share/ebf387404c2f4ad6bbdbd91b318444d2)
+- Loom: [Loom]((https://www.loom.com/share/0524f28eca464fb097b9c347393b10f4))
```

Comment:
- [ ] **[OPTIONAL]** Kindly consider adding a section for the presentation video (I suggest just before or after the live demo section). This will make it more accessible, I had a really hard time founding it 😆 😄